spark/src/test/scala/geotrellis/spark/tiling/ZoomedLayoutSchemeSpec.scala

```diff
@@ -9,7 +9,7 @@ import org.scalatest._
 class ZoomedLayoutSchemeSpec extends FunSpec with Matchers {
   describe("ZoomedLayoutScheme") { 
     it("Cuts up the world in two for lowest zoom level") {
-      val LayoutLevel(_, tileLayout) = ZoomedLayoutScheme().levelFor(1)
+      val LayoutLevel(_, tileLayout) = ZoomedLayoutScheme().levelForZoom(LatLng.worldExtent, 1)
```

Comment:
e.g. I usually have a `val layoutScheme = ZoomedLayoutScheme(256)` defined at the top level of code I'm working with (e.g. the WebMercator image tiler for OAM). If I had to do `val layoutScheme = ZoomedLayoutScheme(WebMercator, 256)`, I think that would be better then having to pass in `WebMercator.worldExtent` for every call to `levelForZoom`